contracts/interfaces/modules/dispute/IDisputeModule.sol

```diff
@@ -181,9 +197,20 @@ interface IDisputeModule {
     /// @param data The data to cancel the dispute
     function cancelDispute(uint256 disputeId, bytes calldata data) external;
 
+    /// @notice Tags a derivative if a parent has been tagged with an infringement tag
+    /// @param parentIpId The infringing parent ipId
+    /// @param derivativeIpId The derivative ipId
+    /// @param parentDisputeId The dispute id that tagged the parent ipId as infringing
+    function tagDerivativeIfParentInfringed(
+        address parentIpId,
+        address derivativeIpId,
+        uint256 parentDisputeId
+    ) external;
+
     /// @notice Resolves a dispute after it has been judged
-    /// @param disputeId The dispute id
-    function resolveDispute(uint256 disputeId) external;
+    /// @param disputeId The dispute
+    /// @param data The data to resolve the dispute
+    function resolveDispute(uint256 disputeId, bytes calldata data) external;
```

Comment:
Is there a recommendation of the data format here?

Reply:
For the current arbitration policy SP - we can pass we don't need to pass anything - just "".

The was added for future-proofing.